server/controllers/request.js

```diff
@@ -1,14 +1,36 @@
-require("dotenv").config();
 const Request = require("../models/Request");
 const asyncHandler = require("express-async-handler");
+const Profile = require("../models/Profile");
+const { getRequestsData } = require("../utils/fetchSitterJobs");
+const mongoose = require("mongoose");
 
 // @route GET /request
 // @desc get requests related to a logged in user or sitter
 // @access Private
 exports.getRequest = asyncHandler(async (req, res) => {
-  const request = await Request.find({ userId: req.user.id });
-  if (request) {
-    res.send(200).json({ success: { message: `get the ${request}` } });
+  const userId = req.user.id;
+
+  const sitterProfile = await Profile.findOne({ userId });
+
+  const sitterProfileId = mongoose.Types.ObjectId(sitterProfile.id);
+  const conditions = {
+    sitterId: sitterProfileId,
+  };
+
+  const accepted = await getRequestsData({ ...conditions, status: "ACCEPTED" });
```

Comment:
Any particular reason why the status is all uppercase? Generally for enums it's best to just keep them camelCase or snake_case so "accepted" will suffice